action.yml

```diff
@@ -0,0 +1,55 @@
+inputs:
+  targets:
```

Comment:
Add an optional arg for the indexes.bolt folder, which can be by default ~/.depviz.

Reply:
what will this be used for ?

We don't want to store the indexes.bolt after the computing of the outputs right ?
So we don't really need to make it custom ?